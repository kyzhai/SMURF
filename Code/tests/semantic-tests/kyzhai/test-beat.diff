1c1,10
< Fatal error: exception Sast.Main_missing
---
> Program: 
> 	SVardef: 
> 		ID: a :: Beat
> 
> 	4...
> Global Scope: 
> 	ID: a :: Beat
> 	ID: print :: Unknown
> 	ID: random :: Unknown
> 
