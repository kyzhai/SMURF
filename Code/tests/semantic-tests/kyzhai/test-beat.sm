a = 4..

b = 1...

main = []
