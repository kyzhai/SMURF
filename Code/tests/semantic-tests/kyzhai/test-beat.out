Program: 
	SVardef: 
		ID: a  :: Beat

	4...
	SMain: []
Global Scope: 
	ID: a  :: Beat
	ID: main  :: Unknown
	ID: print x :: a -> a
	ID: random  :: Int

